docs/conf.py

```diff
@@ -130,7 +129,7 @@ def check_sphinx_version(expected_version):
 
 # General information about the project
 project = setup_cfg['name']
-author = setup_cfg['author']
+author = setup_cfg['authors'][0]['name']
```

Comment:
this will read the first author in the author list; perhaps it should append the email address as `Author Name <address@email.com>`, in which case this line would need to query for the existence of the `email` key